service/src/main/java/org/cbioportal/service/util/AlterationEnrichmentUtil.java

```diff
@@ -160,7 +163,7 @@ public List<AlterationEnrichment> createAlterationEnrichments(Map<String, Pair<L
     
     public long includeFrequencyForSamples(
             List<MolecularProfileCaseIdentifier> molecularProfileCaseIdentifiers,
-            List<T> alterationCountByGenes,
+            List<T> alterationCounts,
```

Comment:
Just wondering, could we change this `T` to `AlterationCountByGene` too?

Reply:
not here I think: profiledCasesCounter.calculate also expects a T (` extends AlterationCountBase`)